pkg/solana/txm/txm_race_test.go

```diff
@@ -84,7 +85,12 @@ func TestTxm_SendWithRetry_Race(t *testing.T) {
 
 	t.Run("delay in rebroadcasting tx", func(t *testing.T) {
 		client := clientmocks.NewReaderWriter(t)
-		// client mock
+		client.On("LatestBlockhash", mock.Anything).Return(&rpc.GetLatestBlockhashResult{
```

Comment:
Like in the other tests, I think we can move this mock to the top level since we return the same thing for all of them